src/windows-emulator/syscalls.cpp

```diff
@@ -166,10 +171,10 @@ namespace
 	}
 
 	NTSTATUS handle_NtQueryValueKey(const syscall_context& c, const handle key_handle,
-	                                const emulator_object<UNICODE_STRING> value_name,
-	                                const KEY_VALUE_INFORMATION_CLASS key_value_information_class,
-	                                const uint64_t key_value_information,
-	                                const ULONG length, const emulator_object<ULONG> result_length)
+		const emulator_object<UNICODE_STRING<EmulatorTraits<Emu64>>> value_name,
```

Comment:
formatting seems off at a few playes. i will do some manual formatting and then introduce some editorconfig or clang-format configs to the project, to make sure formatting is consistent

Reply:
done

Reply:
manual formatting is done. i will open an issue for clang-format/editorconfig